<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectPclOsc</class>
 <widget class="QMainWindow" name="KinectPclOsc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>729</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>255</horstretch>
     <verstretch>255</verstretch>
    </sizepolicy>
   </property>
   <widget class="QVTKWidget" name="qvtk_widget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="pauseCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause Grabber</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="computeNormalsCheckbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute Normals</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="findSHOTdescriptors">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Find SHOT Descriptors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveCloudButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Pointcloud</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="objectNameTextInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Object Name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="matchModelsCheckbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Match Models</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadRawCloudButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Raw Cloud</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="removeNoiseCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Noise</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipTextInput">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>650</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>650</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>650</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portTextInput">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>650</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setOscTargetButton">
    <property name="geometry">
     <rect>
      <x>797</x>
      <y>690</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set OSC Target</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Models Folder</string>
    </property>
   </widget>
   <widget class="QSlider" name="depthThresholdSlider">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>29</width>
      <height>571</height>
     </rect>
    </property>
    <property name="minimum">
     <number>250</number>
    </property>
    <property name="maximum">
     <number>2500</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Downsampling Radius</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="downsamplingRadiusEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>400</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="downsamplingRadiusSlider">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>441</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelsFolderEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseForModelsButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QSlider" name="depthImageThresholdSlider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>29</width>
      <height>331</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="blobs" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>411</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
